spec/queries/district_analytics_query_spec.rb

```diff
@@ -72,11 +77,11 @@
         #
         Timecop.travel(month + 2.months) do
           patients_1.each do |patient|
-            create(:blood_pressure, patient: patient, facility: facility_1)
+            create(:blood_pressure, patient: patient, facility: facility_1, user: user)
           end
 
           patients_2.each do |patient|
-            create(:blood_pressure, patient: patient, facility: facility_2)
+            create(:blood_pressure, patient: patient, facility: facility_2, user: user)
```

Comment:
I assume because the factories were making unique users per bp reading. This is definitely the kind of thing that makes factorybot hard to use efficiently.